src/domain/app/header/header.module.scss

```diff
@@ -1,9 +1,33 @@
-@import 'layout';
-
-.headerWrapper {
-  display: grid;
-  background-color: var(--color-white);
-  position: sticky;
-  top: 0;
-  z-index: 1001;
+@import 'node_modules/hds-design-tokens/lib/breakpoint/breakpoint.scss';
+
+.navigation {
+  .navigationItem {
+    color: var(--navigation-item-color) !important;
+  }
+  .addEventItem {
+    margin-left: auto !important;
+
+    &.hidden {
+      visibility: hidden;
+    }
+
+    svg {
+      margin-right: var(--spacing-xs);
+    }
+  }
+  .navigationItemLabel {
+    display: flex;
+    align-items: center;
+  }
+
+  .languageSelector {
+    @media only screen and (max-width: $breakpoint-m) {
```

Comment:
now would be good chance to add breakpoints.scss file to this project, and use mixins from there, also those breakpoints could be imported from design tokens, see:
https://github.com/City-of-Helsinki/linkedcomponents-ui/blob/develop/src/assets/styles/breakpoints.scss

Reply:
or use the the mixins from layout.scss

Reply:
here is a good example how those mixins are used:

https://github.com/City-of-Helsinki/events-helsinki-ui/blob/develop/src/domain/eventSearch/search.module.scss

Reply:
so 2 options:

1. use breakpoints the same way linkedcomponents-ui uses
2. use the existing mixins from layout.scss (just add breakpoints from design tokens

Reply:
layout.scss seem have a bit different breakpoint values than what comes from design tokens:
`$breakpoints: (
  xs: 576px,
  sm: 768px,
  md: 1024px,
  lg: 1200px,
  xlg: 1600px
);
`
Design tokens:
`$breakpoint-xs: 320px;
$breakpoint-s: 576px;
$breakpoint-m: 768px;
$breakpoint-l: 992px;
$breakpoint-xl: 1248px;`

Also layout.scss wants key value as parameter (xs, sm, md etc...) which does not go together with design tokens. 
So probably I go for same ways as linkedcomponents-ui does (1)

Reply:
introduced new breakpoints.scss, layout.scss can be deprecated

Reply:
This is very good now :)